CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing Guidelines
+
+There are two main ways to contribute to the project &mdash; submitting issues and submitting 
```

Comment:
The "submitting issues" part is missing in this document. Please add it as well.

Reply:
Noted. I have dropped the `submitting changes` section for a `submitting prs` and `submit issues` section in the latest commit. For context, I initially had it in a similar format as https://github.com/Kotlin/kotlinx-io/blob/master/CONTRIBUTING.md.